src/mac/process.rs

```diff
@@ -16,9 +16,11 @@ pub struct Process {
     pub cmd: String,
     /// path to the executable
     pub exe: String,
-    /// pid of the processus
+    /// pid of the process
     pub pid: i64,
-    /// environment of the processus
+    /// pid of the parent process
+    pub parent: i64,
```

Comment:
A process doesn't always have a parent. `Option<i64>` would be more suitable here.